### ISSUE TITLE
Cannot align bars to center

### ISSUE THREAD

**Issue by Landris18** [katspaugh/wavesurfer.js#3395]
## Bug description
Using version 6.4.0 the bars was aligned automatically to center, now in version 7.5.1 it is not aligned automatically to center and the option barAlign can only setted to **top** or **bottom**

## Environment
 - Browser: Firefox

## Minimal code snippet

```
const options: WaveSurferOptions = {
    container: "#wavecontainer",
    waveColor: "#eee",
    progressColor: "#f25046",
    cursorWidth: 0,
    barWidth: 4,
    barRadius: 3,
    hideScrollbar: true,
    height: 70,
    normalize: false,
    dragToSeek: true
};

```

```<Stack width={"100%"} sx={{ cursor: "pointer" }} id={"wavecontainer"} />```

## Expected result

Bars aligned to center

## Obtained result
Bars not aligned to center

## Screenshots
These are the screenshots made in version 6 and 7 showing the difference :

![image](https://github.com/katspaugh/wavesurfer.js/assets/47665507/d8b01b52-8068-403e-b37c-3249e1c204a8)

![image](https://github.com/katspaugh/wavesurfer.js/assets/47665507/79209cde-4693-46d1-908c-07ee49b277a0)

**Comment by katspaugh** [comment]
They are assymetric because the top part represents the left channel and the bottom part the right.

You can specifiy a `splitChannels` option to render them in mono like this:

```js
const options: WaveSurferOptions = {
    // ...,
    splitChannels: [{
      height: 70,
    }, {
      height: 0,
    }],
};

```

**Comment by Landris18** [comment]
When i do splitChannels like you mentionned (i added container because it's required)

```
const options: WaveSurferOptions = {
    container: "#wavecontainer",
    waveColor: "#eee",
    progressColor: "#f25046",
    cursorWidth: 0,
    barWidth: 4,
    barRadius: 3,
    hideScrollbar: true,
    height: 70,
    normalize: false,
    dragToSeek: true,
    splitChannels: [{
        height: 70, container: "#wavecontainer"
    }, {
        height: 0, container: "#wavecontainer"
    }],
};
```
The bars is aligned to center but i got two waveforms. Even if i put only one object in splitChannels i get two waveforms anyway: 

![image](https://github.com/katspaugh/wavesurfer.js/assets/47665507/2bba0262-c486-42f1-aa39-cddfc8e63d2c)

**To be brief, the bars are now aligned to center but i have two waveforms.**

**Comment by katspaugh** [comment]
Oops, looks like the height option is ignored in splitChannels, it's a bug. I've filed an issue for that: https://github.com/katspaugh/wavesurfer.js/issues/3400

**Comment by Landris18** [comment]
@katspaugh please tell me when it's fixed 🙏

**Comment by katspaugh** [comment]
Fixed in 7.5.2. ✅

**Comment by Landris18** [comment]
Thanks
